### PR TITLE
Next 13 App router

### PR DESCRIPTION
**Pull request by helloitsjoe**
## Description

Migrating from `pages` to `app`! https://nextjs.org/docs/app/building-your-application/upgrading/app-router-migration

## How was this tested?

Clicked around and tried to break things

### CODE REVIEW

File: components/Layout.js [hunk 1]
```diff
@@ -5,15 +5,11 @@ import styles from '../styles/Layout.module.css';
 
 const Layout = ({ children }) => {
   return (
-    <>
-      <Head>
-        <title>Runtime Rundown</title>
-        <link rel="icon" href="/favicon.ico" />
-      </Head>
+    <div id="__next">
```
Comment (helloitsjoe) [inline components/Layout.js:8]: This keeps styling consistent, `pages` includes this by default but `app` doesn't

File: components/LogoBanner.js [hunk 2]
```diff
@@ -4,7 +4,7 @@ import styles from '../styles/LogoBanner.module.css';
 
 const LogoBanner = () => {
   return (
-    <Link href="/">
+    <Link href="/" legacyBehavior>
```
Comment (helloitsjoe) [inline components/LogoBanner.js:7]: Link codemod added this, syling is a bit wonky without it

File: app/questions/page.js [hunk 3]
```diff
@@ -1,7 +1,8 @@
-import styles from '../styles/SuggestionsPage.module.css';
-import LineBreak from '../components/LineBreak';
-import LogoBanner from '../components/LogoBanner';
-import ContactForm, { formTypes } from '../components/ContactForm';
+import styles from '../../styles/SuggestionsPage.module.css';
+import LineBreak from '../../components/LineBreak';
+import LogoBanner from '../../components/LogoBanner';
+import ContactForm from '../../components/ContactForm';
+import { formTypes } from '../../components/constants';
```
Comment (helloitsjoe) [inline app/questions/page.js:5]: Had to move this into a `constants` file, I was getting an error when trying to import from a client component:

```
"you cannot dot into a client module from a server component"
```

File: components/Comments.js [hunk 4]
```diff
@@ -1,3 +1,5 @@
+'use client';
```
Comment (helloitsjoe) [inline components/Comments.js:1]: Refs can only be used in client components

File: styles/Header.module.css [hunk 5]
```diff
@@ -44,26 +42,44 @@
...
 }
 
-
 @-webkit-keyframes wiggle {
-  0% {-webkit-transform: rotate(4deg);}
-  25% {-webkit-transform: rotate(-4deg);}
-  50% {-webkit-transform: rotate(6deg);}
-  75% {-webkit-transform: rotate(-2deg);}
-  100% {-webkit-transform: rotate(0deg);}
+  0% {
```
Comment (helloitsjoe) [inline styles/Header.module.css:46]: Prettier 🙄

File: components/Hero.js [hunk 6]
```diff
@@ -22,17 +22,15 @@ const Hero = ({ isShort = false }) => {
               >
                 <div className={styles.profileCard}>
                   <Image
-                    height="75px"
-                    width="75px"
+                    height={75}
```
Comment (helloitsjoe) [inline components/Hero.js:25]: New NextJS Image component had the wrong sizes when `px` are included 🤷‍♂️

**Review by Cooperbuilt — APPROVED** [review]

**Pull request merged**